ci-operator/jobs/openshift/cluster-api-provider-libvirt/openshift-cluster-api-provider-libvirt-master-presubmits.yaml

```diff
@@ -49,200 +49,80 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )actuator-pkg-staleness,?($|\s.*)
   - agent: kubernetes
-    always_run: true
```

Comment:
@Fedosin This should be true because we always want this job to be run for each PR.